scripts/import/import_EPMC.pl

```diff
@@ -268,10 +268,21 @@ sub import_citations{
         }
         else{
             ## add new publication
+	    my $new_title = $title;
```

Comment:
Could a regexp be used to remove the last characters preceded by a space instead of splitting and joining?

Reply:
I tried that, however if title is very long the last occurrence of a space is after the 300 characters. Splittting and joining was the only way I found to add characters until 300 characters and ignore everything else.

Reply:
The suggestion was to use a regexp to remove the last characters preceded by a space instead of splitting and joining after the truncation to 296 characters.  As is the array works, this is fine.  A ticket can be added to review use of regexp for truncation as done in trim_value_string in BasePhenotypeAnnotation.pm